app/Http/Controllers/UsersController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Gladiator\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Gladiator\Models\User;
+use Gladiator\Models\WaitingRoom;
+use Gladiator\Http\Requests\UserRequest;
+
+class UsersController extends Controller
+{
+    /**
+     * Create new UsersController instance.
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+        $this->middleware('role:admin,staff');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $admins = User::where('role', '=', 'admin')->get();
+        $staff = User::where('role', '=', 'staff')->get();
+        $contestants = User::where('role', '=', null)->get();
```

Comment:
Yeah, I need to go back in and add pagination to it and likely separate out the Admin and Staff users on one page and contestant users on another.